### PR TITLE
Feat/transaction

### PR DESCRIPTION
**Pull request by Anon-136**
Screen Shot 2565-03-01 at 21 41 34

### CODE REVIEW

**Review by paphonb — APPROVED** [review]

File: app/withdrawal/mutations/withdraw.ts [hunk 1]
```diff
@@ -11,37 +11,29 @@ import { Withdrawal } from '../validations'
 const withdraw = resolver.pipe(
   resolver.zod(Withdrawal),
   resolver.authorize(),
-  async ({ bank, account, amount }, ctx: Ctx) => {
-    if (!ctx.session.userId) throw new AuthorizationError()
+  async (input, { session: { userId } }: Ctx) => {
+    if (!userId) throw new AuthorizationError()
```
Comment (paphonb) [inline app/withdrawal/mutations/withdraw.ts:15]:
```suggestion
  async (input, { session: { userId } }) => {
```

Comment (paphonb) [reply app/withdrawal/mutations/withdraw.ts:15]: I copied it too lol

Comment (paphonb) [reply app/withdrawal/mutations/withdraw.ts:15]: Fixed it in #28 but didn't touch this file as I was worried it might conflict.

Comment (paphonb) [reply app/withdrawal/mutations/withdraw.ts:15]: Turns out there's already a conflict 😂

Comment (Anon-136) [reply app/withdrawal/mutations/withdraw.ts:15]: Oh nvm got it

**Pull request merged**